scripts/build-image.sh

```diff
@@ -96,13 +96,14 @@ else
     echo "Unsupported CPU architecture: $(uname -m)"
     return 1
 fi
-/usr/bin/wget "https://github.com/mikefarah/yq/releases/latest/download/yq_linux_$YQ_ARCH"
-/usr/bin/wget https://github.com/mikefarah/yq/releases/latest/download/checksums
-/usr/bin/wget https://github.com/mikefarah/yq/releases/latest/download/checksums_hashes_order
-/usr/bin/wget https://github.com/mikefarah/yq/releases/latest/download/extract-checksum.sh
+/usr/bin/wget "https://github.com/mikefarah/yq/releases/latest/download/yq_linux_$YQ_ARCH" -O "yq_linux_$YQ_ARCH"
```

Comment:
May I ask if this is for style consistency since this ends up producing the same output binary name? (same for below)

Reply:
Specifying the output with the `-O` option (upper-case O), will make wget to overwrite the file if it is there. Without this option, wget would be appending `.1`, `.2`, ... to the filename after the first run to prevent overwriting existing files.